sci-biology/spm/spm-8.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2013 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+DESCRIPTION="Analysis of brain imaging data sequences for Octave or Matlab"
+HOMEPAGE="http://www.fil.ion.ucl.ac.uk/spm/"
+URI_BASE="http://www.fil.ion.ucl.ac.uk/spm/download/restricted/idyll/"
+SRC_URI="${URI_BASE}${PN}${PV}.zip"
+
+LICENSE="GPL-2+"
+SLOT="8"
```

Comment:
SPM has a weird versioning scheme, whereby different versions are (almost) treated as different programs. For instance, the version number is part of the executable name. I thought slots would be a good Gentoo analogue for that, and would also allow users to have multiple versions of it installed. Most work is still done on SPM-8, and users would need it to reproduce past analyses, but newer or experimental projects would likely be run on SPM-12.